torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -266,6 +266,14 @@ def _deduce_value(self, node: torch.fx.Node):
             if isinstance(out, torch.Tensor) and out.numel() == 1:
                 return out
 
+        # handle device_put op
+        if (
+            node.op == "call_function"
+            and hasattr(node.target, "overloadpacket")
```

Comment:
nit: you can just do `node.target == prims.device_put.default`:

Reply:
Updated, thanks.